src/app/(landing)/@about/about.module.scss

```diff
@@ -0,0 +1,27 @@
+@use '../../../styles/colors';
+@use '../../../styles/mixins';
+
+.frame {
+    background-color: colors.getColor(secondary, 50);
+  }
+
+  .container {
```

Comment:
Use the flex component's props

Reply:
got rid of styles.textbox, used Flex component align